jobs/kola-openstack.Jenkinsfile

```diff
@@ -41,6 +41,8 @@ properties([
 
 currentBuild.description = "[${params.STREAM}][${params.ARCH}] - ${params.VERSION}"
 
+def variant = stream_info.variant ? "--variant ${stream_info.variant}" : ""
```

Comment:
I suppose you meant to define `stream_info` here?

Reply:
Fixed